fhd_output/fft_filters/filter_uv_hanning.pro

```diff
@@ -1,4 +1,5 @@
-FUNCTION filter_uv_hanning,image_uv,name=name,weights=weights,filter=filter,return_name_only=return_name_only,_Extra=extra
+FUNCTION filter_uv_hanning,image_uv,obs=obs,psf=psf,params=params,name=name,$
+    weights=weights,filter=filter,return_name_only=return_name_only,_Extra=extra
```

Comment:
I think you don't have to change the call signature, since if `obs`, `psf`, and `params` are passed in they will get bundled in `extra`.